ee/codegen/src/__test__/__snapshots__/workflow.test.ts.snap

```diff
@@ -2,9 +2,9 @@
 
 exports[`Workflow > write > graph > should be correct for a basic conditional node case 1`] = `
 "from vellum.workflows import BaseWorkflow
-from .nodes.conditional_node import ConditionalNode
```

Comment:
I was hoping for no snapshot changes, but something about how we're now doing references changed the import order for these pre-isort imports